### ISSUE TITLE
Unable to create polls after installation (php7)

### ISSUE THREAD

**Issue by hidrarga** [YunoHost-Apps/opensondage_ynh#44]
I was unable to create polls (either classic or date poll) after a fresh installation, with php7.0. I get a blank page with HTTT 500 error. In the log, the following error occurs : 

```
FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Call to undefined function utf8_encode() in /var/www/opensondage/create_classic_poll.php:170
```

I was able to fix the issue by installing php7.0-xml.

Would it be possible to add this dependency when php7 is in use ? Thanks !

**Comment by zamentur** [comment]
On my yunohost php7.0-xml is installed because php7.0-fpm (a yunohost requirements).

So it's strange you needed to setup it !

**Comment by hidrarga** [comment]
I've just checked, and php7.0-xml was not a Yunohost requirements on my server. I'm running Yunohost 3.2.2 on Debian stretch. 

Anyways, not all Yunohost installs are the same, and it wouldn't hurt to add this requirement for this app, right ?

**Comment by zamentur** [comment]
Are you on an arm board ?

**Comment by spctrlth** [comment]
No, this issue is encountered on a Hetzner server CX11

**Comment by kay0u** [comment]
Closed by #51
